### ISSUE TITLE
App is missing

### ISSUE THREAD

**Issue by SupermansZ24** [deathgrindfreak/clashcallermobile#2]
The link to the Google Play Store is down. Is there a possibility to re-upload it with the latest commits to some where else?

Cheers

**Comment by deathgrindfreak** [comment]
Yeah, sorry about that.  Google's being kind of an ass about taking it down constantly because of "impersonation" issues.  But I'll try to put it up soon (which is a fun process in and of itself) and will close this issue when I do.
